src/objc/classes.rs

```diff
@@ -675,3 +676,29 @@ impl ObjC {
         }
     }
 }
+
+pub fn class_getName(env: &mut Environment, class: Class) -> ConstPtr<u8> {
+    let class_name = class_getName_inner(&mut env.objc, class);
+    env.mem
+        .alloc_and_write_cstr(class_name.as_bytes())
+        .cast_const()
+}
+
+/// Internal method that gets class name as a &str.
+pub fn class_getName_inner(objc: &mut ObjC, class: Class) -> &str {
```

Comment:
This inner function should probably be a method on `ObjC`. It could be called `get_class_name` I suppose.